src/shared/styles/index.css

```
@@ -0,0 +1,55 @@
+@import 'modern-normalize';
+
+/* Глобальные переменные */
+:root {
+  --background-color-primary: #e7e7e7;
+  --background-color-secondary: rgba(0, 86, 179, 1);
+
+  --font-family-primary: 'Roboto', sans-serif;
+  --font-family-secondary: 'Inter', sans-serif;
+
+  --font-color-primary: rgba(0, 0, 0, 0.87);
+  --font-color-secondary: rgba(0, 0, 0, 0.6);
+
+  --font-color-accent: rgba(25, 118, 210, 1);
+  --icon-color: rgba(0, 0, 0, 0.56);
+}
+
+/* Основные стили */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  color: var(--font-color-primary);
+  font-family: var(--font-family-primary);
+  font-weight: 400;
+}
+
+main {
+  background-color: var(--background-color-primary);
+}
```

```suggestion
body {
  margin: 0;
  padding: 0;
  color: var(--font-color-primary);
  font-family: var(--font-family-primary);
  font-weight: 400;
  background-color: var(--background-color-primary);
}
```
Предлагаю перенести `background-color` в тэг `body`